framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useState } from 'react'
-import { CreatableSelect } from 'chakra-react-select'
-import type { Option } from '@northlight/ui'
+import { ActionMeta, CreatableSelect } from 'chakra-react-select'
+import type { Option, SingleValue } from '@northlight/ui'
 import { PlusSolid } from '@northlight/icons'
 import { Icon } from '@chakra-ui/react'
+import { customSelectStyles } from '../../theme/components/select/custom-select'
```

Comment:
??

Reply:
this is to add the same styling to createable select as normal select, such as blue dropdown icon to open it